src/Maestro/DependencyUpdater/DependencyUpdater.cs

```diff
@@ -203,6 +207,63 @@ select new
             }
         }
 
+        /// <summary>
+        ///     Update Longest Build Path table once a day for each channel at midnight
+        /// </summary>
+        /// <param name="cancellationToken"></param>
+        /// <returns></returns>
```

Comment:
I'd remove the empty params and returns elements. :-)  Also, the comment is just a restatement of the name of the method and the attribute, so it's not adding much... I'd probably remove the whole comment block